sapi/fpm/tests/tester.inc

```diff
@@ -1253,6 +1253,37 @@ class Tester
         return true;
     }
 
+    public function expectLogConfigOptions(array $options)
+    {
+        $configOptions = $this->getConfigOptions();
+        foreach ($options as $name => $value) {
+            if (!isset($configOptions[$name])) {
+                return $this->error("Expected config option: {$name} = {$value} but {$name} is not set");
+            }
+            if ($configOptions[$name] != $value) {
+                return $this->error(
+                    "Expected config option: {$name} = {$value} but got: {$name} = {$configOptions[$name]}"
+                );
+            }
+        }
+
+        return true;
+    }
+
+    private function getConfigOptions()
```

Comment:
please add doc block comment so it's a bit more consistent with other functions.